### PR TITLE
add changeProduct

### PR DESCRIPTION
**Pull request by R-Schreyer**

### CODE REVIEW

File: frontend/src/components/Order/ViewOrders.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.order-display {
+
+    margin: 20px auto;!* Zentrieren der Tabelle horizontal *!
+    border-collapse: collapse; !* Zusammenführen von Zellengrenzen *!
+}
+
+.head-line td {
+    text-align: left; !* Text von links nach rechts ausrichten *!
+    direction: rtl; !* Zelleninhalt von rechts nach links anordnen *!
+}*/
```
Comment (ReebalSami) [inline frontend/src/components/Order/ViewOrders.css:11]: Do you need this comment? 
```suggestion
```

File: frontend/src/components/Order/ViewOrders.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    border-collapse: collapse; /* Zusammenführen von Zellengrenzen */
+}
+
+.head-line td {
+    text-align: left; /* Text von links nach rechts ausrichten */
+    direction: rtl; /* Zelleninhalt von rechts nach links anordnen */
+}
+
+.order-display td {
+    padding-right: 8px; /* Hinzufügen von Abstand um den Text in den Zellen */
```
Comment (ReebalSami) [inline frontend/src/components/Order/ViewOrders.css:27]: same here? :)
```suggestion
    text-align: center;
}

.order-display {
    margin: 20px auto; 
    border-collapse: collapse; 
}

.head-line td {
    text-align: left; 
    direction: rtl;
}

.order-display td {
    padding-right: 8px; 
```

File: frontend/src/components/Order/ViewOrders.tsx [hunk 3]
```diff
@@ -17,8 +18,30 @@ export default function ViewOrders({customers}: Readonly<ViewOrdersProps>) {
...
+                    <tr key={order.id}>
+                        <td>{order.id}</td>
+                        <td>{order.price}</td>
+                    </tr>
+                ))}
+            </table>
+        </div>
+        /*<div>
+            <h1>Bestellungen für {customer.firstname} {customer.lastname}</h1>
+            <div className="button-container">
```
Comment (ReebalSami) [inline frontend/src/components/Order/ViewOrders.tsx:41]: Comment is not needed.

File: frontend/src/components/Product/ChangeProduct.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+export default function ChangeProduct(props: Readonly<ProductChangeProps>) {
+    const [productName, setProductName] = useState<string>(props.product.productName);
+    const [category, setCategory] = useState(props.product.category);
+    const [pricePerPiece, setPricePerPiece] = useState<number>(props.product.pricePerPiece);
+    const [producer, setProducer] = useState(props.product.producer);
+    const [quantity, setQuantity] = useState(props.product.quantity);
+    const handlePriceChange = (event: ChangeEvent<HTMLInputElement>) => {
+        const newValue = parseFloat(event.target.value);
+        // Überprüfe, ob newValue eine gültige Zahl ist
```
Comment (ReebalSami) [inline frontend/src/components/Product/ChangeProduct.tsx:17]: same here :)

File: frontend/src/components/Product/ChangeProduct.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+        if (!isNaN(newValue)) {
+            // Wenn die Eingabe eine gültige Zahl ist, setzen Sie den Zustand auf diese Zahl
```
Comment (ReebalSami) [inline frontend/src/components/Product/ChangeProduct.tsx:20]: and here :)

**Review by ReebalSami — CHANGES_REQUESTED** [review]
It would be great, if you could remove the unneeded comments. For the most of them I left you suggestions, but there were comments, that has been updated with this pull request. those need to be changed by you. :)

File: frontend/src/components/Product/ChangeProduct.tsx (hunk 5 above)
Comment (R-Schreyer) [reply frontend/src/components/Product/ChangeProduct.tsx]: thx, think now they should be removed

**Review by MarcellNF — APPROVED** [review]
LGTM!:)

**Pull request merged**